python/ray/experimental/sgd/pytorch/pytorch_utils.py

```diff
@@ -1,107 +0,0 @@
-from __future__ import absolute_import
```

Comment:
happy to rename this to `utils.py` as you have done; can you make sure all references and imports of `ray.experimental.sgd.pytorch.pytorch_utils` point correctly to `ray.experimental.sgd.pytorch.utils`?

Reply:
should be fixed